src/containers/BooksForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const BookForm = () => (
+  <form>
+    <label htmlFor="title">
+      Title
+      <input type="text" name="title" id="title" />
+      <br />
+    </label>
+    <label htmlFor="category">
+      Category
+      <select id="category" name="category">
+        <option value="Action">Action</option>
+        <option value="Biography">Biography</option>
+        <option value="Horror">Horror</option>
+        <option value="History">History</option>
```

Comment:
Then use map in here.

It could be like 👇 
```
      <select required>
        {categories.map(category => (
          <option key={category}>{category}</option>
        ))}
      </select>
```